lib/src/main/java/io/ably/lib/transport/ITransport.java

```diff
@@ -51,6 +51,7 @@ public TransportParams() {
 
 		public Param[] getConnectParams(Param[] baseParams) {
 			List<Param> paramList = new ArrayList<Param>(Arrays.asList(baseParams));
+			paramList.add(new Param("v", HttpUtils.X_ABLY_VERSION_VALUE));
```

Comment:
See https://github.com/ably/ably-java/blob/master/lib/src/test/java/io/ably/lib/test/rest/HttpHeaderTest.java.  That test prevents a regression where the header is no longer sent mistakenly, or incorrectly sent.  Same thing no?